hw/xbox/xbox.c

```diff
@@ -502,6 +518,11 @@ static inline void xbox_machine_initfn(Object *obj)
     object_property_set_description(obj, "eeprom",
                                     "Xbox EEPROM file", NULL);
 
+    object_property_add_str(obj, "avpack", machine_get_avpack,
+                            machine_set_avpack, NULL);
+    object_property_set_description(obj, "avpack",
+                                    "Xbox video connector: standard (default), none, scart, svideo, vga, rfu, hdtv", NULL);
```

Comment:
Consider ordering these by something meaningful. I'd put "none" first or last.

nit: Consider adopting same order in your mapping function.